code/game/objects/explosion.dm

```diff
@@ -31,33 +31,33 @@ proc/explosion(turf/epicenter, devastation_range, heavy_impact_range, light_impa
 // Calculate far explosion sound range. Only allow the sound effect for heavy/devastating explosions.
 
 // 3/7/14 will calculate to 80 + 35
-		var/far_dist = 0
-		far_dist += heavy_impact_range * 5
-		far_dist += devastation_range * 20
+		var/volume = 10 + (power * 20)
 		var/frequency = get_rand_frequency()
+		var/closedist = round(max_range + world.view - 2, 1)
 		for(var/mob/M in player_list)
 			// Double check for client
 			if(M && M.client)
 				var/turf/M_turf = get_turf(M)
 				if(M_turf && M_turf.z == epicenter.z)
 					var/dist = get_dist(M_turf, epicenter)
-					// If inside the blast radius + world.view - 2
-					if(dist <= round(max_range + world.view - 2, 1))
-						M.playsound_local(epicenter, get_sfx("explosion"), 100, 1, frequency, falloff = 5) // get_sfx() is so that everyone gets the same sound
-
-						//You hear a far explosion if you're outside the blast radius. Small bombs shouldn't be heard all over the station.
-
-					else if(dist <= far_dist)
-						var/far_volume = Clamp(far_dist, 30, 50) // Volume is based on explosion size and dist
-						far_volume += (dist <= far_dist * 0.5 ? 50 : 0) // add 50 volume if the mob is pretty close to the explosion
-						M.playsound_local(epicenter, 'sound/effects/explosionfar.ogg', far_volume, 1, frequency, falloff = 5)
-
-		var/close = range(world.view+round(devastation_range,1), epicenter)
-		// to all distanced mobs play a different sound
-		for(var/mob/M in world) if(M.z == epicenter.z) if(!(M in close))
-			// check if the mob can hear
-			if(M.ear_deaf <= 0 || !M.ear_deaf) if(!istype(M.loc,/turf/space))
-				M << 'sound/effects/explosionfar.ogg'
+					if (M.ear_deaf <= 0 || !M.ear_deaf)//Dont play sounds to deaf people

```

Comment:
Missing a check for the user being in space.

Sound requires a medium to travel by, and being in space should mean you don't hear the boom. This was covered by the `if(!istype(M.loc,/turf/space)` condition in the outdated version.

**Unless** `playsound_local()` takes care of it.